Terminal.Gui/Views/Slider.cs

```diff
@@ -59,7 +59,7 @@ private void SetInitialProperties (
         // BUGBUG: This should not be needed - Need to ensure SetRelativeLayout gets called during EndInit
         Initialized += (s, e) => { SetContentSize (); };
 
-        SubviewLayout += (s, e) => { SetContentSize (); };
+        SubViewLayout += (s, e) => { SetContentSize (); };
```

Comment:
what is difference between Layout and LaidOut?

Reply:
`Layout` == Layout is starting
`LaidOut` == Layout is complete

I struggled with this naming given the standard guidance on naming that says don't use "Starting/Completed".

Reply:
Hmn... ok lets go with it.